cot/src/db.rs

```diff
@@ -607,7 +670,65 @@ impl Database {
             data.update_from_db(row, &auto_col_ids)?;
         }
 
-        debug!("Inserted row");
+        if update {
+            debug!("Inserted or updated row");
+        } else {
+            debug!("Inserted row");
+        }
```

Comment:
I think it could be beneficial to include more metadata here, maybe table name and row id?

Reply:
No better time like now to start using `tracing` crate `span`s and whatnot!